tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -386,10 +412,10 @@ const syncAndWaitForFailure = async () => {
   await (await syncRequired()).waitForDisplayed({ timeout: 20000 });
 };
 
-const closeReloadModal = async (shouldUpdate = false, timeout = 5000) => {
+// Increase timeout to 15secs as it sometimes takes a while for the reload modal to display
+const closeReloadModal = async (shouldUpdate = false, timeout = 15000) => {
   try {
     shouldUpdate ? await modalPage.submit(timeout) : await modalPage.cancel(timeout);
-    await modalPage.checkModalHasClosed();
```

Comment:
Why did you delete this verification? If the method `checkModalHasClosed()` has something that is not working it will be better to adapt it instead of deleting the verification